packages/broker/src/plugins/operator/MaintainOperatorValueService.ts

```diff
@@ -1,81 +1,49 @@
 import { Logger, scheduleAtInterval } from '@streamr/utils'
 import { MaintainOperatorValueHelper } from './MaintainOperatorValueHelper'
 import { OperatorServiceConfig } from './OperatorPlugin'
-import { BigNumber } from 'ethers'
 
 const logger = new Logger(module)
 
-const CHECK_VALUE_INTERVAL = 1000 * 60 * 60 * 24 // 1 day
-const ONE_ETHER = BigInt(1e18)
+const DEFAULT_CHECK_VALUE_INTERVAL_MS = 1000 * 60 * 60 * 24 // 1 day
 
 export class MaintainOperatorValueService {
-    private penaltyLimitFraction: bigint
+    private readonly withdrawLimitSafetyFraction: bigint
+    private penaltyLimitFraction?: bigint
     private readonly helper: MaintainOperatorValueHelper
     private readonly abortController: AbortController
+    private readonly checkIntervalMs: number
```

Comment:
(nitpick) To be in line with rest of code base, we've usually used "inMs" suffix. 
```suggestion
    private readonly checkIntervalInMs: number
```

Reply:
Changed